libr/core/cmd_type.c

```diff
@@ -1252,7 +1253,7 @@ static int cmd_type(void *data, const char *input) {
 					}
 				} else {
 					char *error_msg = NULL;
-					char *out = r_parse_c_file (core->anal, filename, &error_msg);
+					char *out = r_parse_c_file (core->anal, filename, dir, &error_msg);
```

Comment:
random though #2 - what if we want to add more includedirs? or maybe other options to the parser? the rparse api needs a refactor. and this must be added to the list of things to do at some point

Reply:
Can we already set more than one includedir in the eval var, I mean, is that already implemented?
I believe it shouldn't be way too hard to pass a dirlist as the argument, to be parsed by tcc in order to add many include directories.

Reply:
wdyt?